Formula/cassandra-reaper.rb

```diff
@@ -24,14 +25,14 @@ def install
 
   service do
     run opt_bin/"cassandra-reaper"
-    environment_variables JAVA_HOME: Formula["openjdk@8"].opt_prefix
+    environment_variables JAVA_HOME: Language::Java.java_home("11")
```

Comment:
Given that we need to set this here and in the `test` block: I wonder if we shouldn't be wrapping `cassandra-reaper` in an env script instead.

Reply:
Probably as default is just `java` on `PATH`
```bash
# Use JAVA_HOME if set, otherwise look for java in PATH
if [ -n "$JAVA_HOME" ]; then
    # Why we can't have nice things: Solaris combines x86 and x86_64
    # installations in the same tree, using an unconventional path for the
    # 64bit JVM.  Since we prefer 64bit, search the alternate path first,
    # (see https://issues.apache.org/jira/browse/CASSANDRA-4638).
    for java in "$JAVA_HOME"/bin/amd64/java "$JAVA_HOME"/bin/java; do
        if [ -x "$java" ]; then
            JAVA="$java"
            break
        fi
    done
else
    JAVA=java
fi
```